app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  # scope :ordered_most_recent, -> { order(created_at: :desc)}
+  default_scope { order(created_at: :desc)}
```

Comment:
The default scope here could be risky. What were your thoughts, priorities adding this. I think I'd prefer the named scope. Definitely better than the class method, though. This is okay.